src/phoenix/server/app.py

```diff
@@ -302,8 +302,48 @@ async def api_docs(request: Request) -> Response:
     return get_swagger_ui_html(openapi_url="/schema", title="arize-phoenix API")
 
 
-def create_app(
+class SessionFactory:
+    def __init__(
+        self,
+        session_factory: Callable[[], AsyncContextManager[AsyncSession]],
```

Comment:
Feels strange to have an input argument that is the same as the name of the class.